helloextend-protection/admin/helloextend_logger_admin.php

```
@@ -46,7 +46,8 @@ function helloextend_logger_load_log_table_scripts()
 
 function helloextend_logger_log_table_scripts()
 {
-	$lastmodtime= filemtime(HELLOEXTEND_LOGGER_URI . 'css/helloextend_logger.css');
+    $css_path = HELLOEXTEND_LOGGER_DIR . 'css/helloextend_logger.css';
```

PHPMD (2.15.0)

The variable $css_path is not named in camelCase. (Controversial Rules)

(CamelCaseVariableName)